server/views/pages/plan.njk

```diff
@@ -110,13 +111,15 @@
             {% endif %}
 
             {% if data.plan.agreementStatus === 'AGREED' %}
-                <p class="govuk-body">{{ locale.agreePlan }}</p>
+{#                {% if data.plan.updatedDate > data.plan.agreementDate %}to be finished at a later date#}
```

Comment:
can you replace the "to finished..." text with `TODO: SP2-836` please? IntelliJ will auto-detect the TODO and it means we can search the codebase for incomplete stuff like this.